eras/conway/impl/src/Cardano/Ledger/Conway/Rules/GovCert.hs

```diff
@@ -235,7 +235,15 @@ conwayGovCertTransition = do
           { vsDReps =
               Map.insert
                 cred
-                (DRepState (addEpochInterval cgceCurrentEpoch ppDRepActivity) mAnchor ppDRepDeposit)
+                ( DRepState
+                    ( computeDRepExpiry
+                        ppDRepActivity
+                        cgceCurrentEpoch
+                        (vState ^. vsNumDormantEpochsL)
+                    )
```

Comment:
That is a change in behavior, which we can't have without a hardfork

Reply:
Done